extensions/wikibase/module/scripts/schema-alignment.js

```diff
@@ -30,7 +30,29 @@ THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
 OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
  */
-
+function returnServiceLogo(reconConfig){
+  var serviceUrl = null;
+  var service = null;
+  var serviceLogo=null;
+  if (reconConfig) {
+    serviceUrl =reconConfig.service;
+  }
+  if (serviceUrl) {
+    service = ReconciliationManager.getServiceFromUrl(serviceUrl);
+  }
+  if(service) {
+    serviceLogo=service.logo;
+  }
+  try {
+    if(new URL(serviceLogo)){
```

Comment:
I think the `if` statement is not really serving a purpose here: the error handling is happening in the catch block already. So it should be enough to have something like `new URL(serviceLogo);`